web/index.html

```diff
@@ -645,9 +777,20 @@ <h3>Used in</h3>
                         console.error('Error loading metadata:', error);
                     });
 
+                fetch('bookmarklet.js')
+                    .then(response => { return response.text() })
+                    .then(bookmarkletText => {
+                        this.bookmarkletData = bookmarkletText.replace('HELPER_URL', HELPER_PATH);
+                    })
+                    .catch(error => {
+                        console.error('Error loading bookmarklet:', error);
+                    });
+
                 addEventListener("popstate", (event) => this.navigateTo(getQuery().e || ""));
-                this.saveData = (getQuery().s || localStorage.getItem("saveData") || "").split(",").filter(x => x)
-                localStorage.setItem("saveData", this.saveData.join(","))
+                this.saveDataArray = (getQuery().s || localStorage.getItem("saveData") || "").split(",").filter(x => x);
+                this.saveData = new Set(this.saveDataArray);
```

Comment:
Not much different, just converting the saved save data to a set for better read times

Reply:
Do we have this.saveDataArray and this.saveData now? Can we remove one or make one computed?

Reply:
Made the saveData set computed. We can't remove either fully as they are both used very often for their different advantages.